tidyblocks/gui.js

```diff
@@ -74,7 +74,7 @@ const setUpBlockly = () => {
     }
   )
 
-  TidyBlocksWorkspace.addChangeListener(Blockly.Events.disableOrphans)
+  //TidyBlocksWorkspace.addChangeListener(Blockly.Events.disableOrphans)
```

Comment:
Whoops: why disable orphans, and why is this in this PR? (It isn't documentation...)